jekyll/_cci2/android-machine-image.adoc

```diff
@@ -141,6 +141,7 @@ jobs:
           # run in parallel with the emulator starting up, to optimize build time
           name: Run assembleDebugAndroidTest task
           command: |
+            export TERM=dumb
```

Comment:
out of curiosity, is this required? is there a possibility this might confuse users?

Reply:
As I said in the commit message:

> Without this TERM= setting we get errors of the form
>
>    Could not open terminal for stdout: could not get termcap entry
> 
> From the "Run assembleDebugAndroidTest task" step.